great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -177,14 +189,35 @@ def get_batch_list_from_batch_request(
         return batch_list
 
 
+_ASSET_MODELS = _generate_data_asset_models(
```

Comment:
@Kilo59 Per above -- what would be contained in `_ASSET_MODELS` (looking for examples).  Thanks.